Remotion/ObjectBinding/Web/UI/Controls/Validation/ValidationFailureHandlingContext.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Text;
+using Remotion.ObjectBinding.Web.UI.Controls.BocListImplementation;
+
+namespace Remotion.ObjectBinding.Web.UI.Controls.Validation
+{
+  public class ValidationFailureHandlingContext
+  {
+    public IBocList BocList { get; }
+
+    public IBocListValidationFailureRepository ValidationFailureRepository { get; }
+
+    public string CompoundedErrorMessage => _sb.ToString();
```

Comment:
I personally would use a method for this functionality. This makes it clearer that there is some kind of computation behind it and it also hides it a bit more from the user.